src/html.rs

```diff
@@ -326,11 +326,11 @@ where
     }
 
     // TODO Consider to use &Node instead of Element as parent
-    /// Mounts elements in place of previous node.
+    /// Mounts elements in place of previous node (ancestor).
     pub fn mount_in_place(
         mut self,
         element: Element,
-        obsolete: Option<VNode<CTX, COMP>>,
```

Comment:
this item gets passed down and is currently called 3 _different_ things:
- obsolete
- opposite
- ancestor

Ancestor is the most clear to me so I renamed all of them that.